### PR TITLE
Add config.yml request to issue template

### PR DESCRIPTION
**Pull request by Benaiah**
**- Summary**

Add config.yml request to issue template

**- Test plan**

N/A

**- Description for the changelog**

Add config.yml request to issue template

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -29,3 +29,5 @@ You do NOT have to include this information if this is a FEATURE REQUEST
 
 **- Please mention your CMS, node.js, and operating system version.**
 <!-- You can find the CMS version by checking your web browser's developer tools console while in the CMS. -->
+
+**- Please link or paste your `config.yml` below if applicable.**
```
Comment (tech4him1) [inline .github/ISSUE_TEMPLATE.md]: Just one thing, it always makes me wonder a little bit when an issue template says "if applicable", unless they explain when it is needed in more detail.

Comment (Benaiah) [reply .github/ISSUE_TEMPLATE.md]: Yeah, I'm just worried that people who don't have (or aren't aware of) the `config.yml` may decide not to file an issue if they think this is required. I'd welcome suggestions for rephrasing this.

Comment (tech4him1) [reply .github/ISSUE_TEMPLATE.md]: "if possible"? that sounds pretty weird, though.

Maybe for the last two questions, we should put something to the effect of: "The following information is not required, but please provide as much as possible to help us reproduce your issue."

**Comment by erquhart** [comment]
@tech4him1 pushed your update, makes sense.

**Comment by verythorough** [comment]
Deploy preview ready!

Built with commit 0ce359bba0ca5eed9de96fd622f66d57ac61f66a

https://deploy-preview-773--netlify-cms-www.netlify.com

**Comment by verythorough** [comment]
Deploy preview ready!

Built with commit 0ce359bba0ca5eed9de96fd622f66d57ac61f66a

https://deploy-preview-773--cms-demo.netlify.com

**Pull request merged**